cluster/gce/util.sh

```diff
@@ -873,7 +873,7 @@ function construct-windows-kubelet-flags {
   flags+=" --logtostderr=false"
 
   # Configure the file path for host dns configuration
-  flags+=" --resolv-conf=${WINDOWS_CNI_CONFIG_DIR}\hostdns.conf"
+  flags+=" --resolv-conf=${WINDOWS_CNI_DIR}\hostdns.conf"
```

Comment:
is it cni at all? it's about DNS.

Reply:
It seems not a CNI thing, it is just network related.

We can organize the directory structure better, currently the cni directory is the only place containing some network stuff. :)

Reply:
ok, let's organize it later. thanks.

Reply:
For windows, dns config is through cni. We don't have a network directory and would be nice to have one. 
/lgtm